src/components/multiselect.vue

```diff
@@ -377,6 +378,12 @@ const toggleAfterEnter = ({ key }) => {
   if (key === 'Enter') $toggle.value.dropdown('toggle');
 };
 
+const apply = () => {
+  searchValue.value = '';
+  emitIfChanged();
+  $dropdown.value.removeClass('open');
```

Comment:
is there any other way to do this?

Reply:
I think we can call the [`.dropdown()`](https://getbootstrap.com/docs/3.4/javascript/#dropdowns-methods) method from the Bootstrap plugin:

```suggestion
  $dropdown.value.dropdown('toggle');
```